adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -80,6 +79,8 @@ class TransformArgs extends Args4jBase with ADAMSaveAnyArgs with ParquetArgs {
   var forceLoadIFastq: Boolean = false
   @Args4jOption(required = false, name = "-force_load_parquet", usage = "Forces Transform to load from Parquet.")
   var forceLoadParquet: Boolean = false
+  @Args4jOption(required = false, name = "-as_regular_file", usage = "Saves OUTPUT as regular file instead of using Hadoop OutputFormat")
```

Comment:
This help message seems a bit inaccurate to me, because `transform` will continue to use an `OutputFormat`.  I'd reword it.